src/PageContent/Calendar/DialogContent.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import './DialogContent.css';
+import DelivererGroupContent from './DelivererGroupContent';
+
+class DialogContent extends Component {
+    render() {
+        return (
+            <div>
+                <DelivererGroupContent
+                    accountType={this.props.accountType}
+                    futureEvent={this.props.futureEvent}
+                    delivererGroup={this.props.delivererGroup}
```

Comment:
same here, don't need to pass it down separately, it's included in `this.props.delivery`